src/main/java/com/mraof/minestuck/client/util/MSKeyHandler.java

```diff
@@ -91,15 +92,27 @@ public static void onKeyInput(InputEvent event)	//This is only called during the
 	@SubscribeEvent
 	public static void onTick(TickEvent.ClientTickEvent event)
 	{
+
 		if(InputMappings.isKeyDown(Minecraft.getInstance().getMainWindow().getHandle(), captchaKey.getKey().getKeyCode()) && !captchaKeyPressed) {
 
 			Screen screen = Minecraft.getInstance().currentScreen;
-			if(screen instanceof ContainerScreen<?> && screen.getFocused() == null && !(screen instanceof CreativeScreen))
```

Comment:
forgot to leave a comment in the code, so I'll mention it here instead: `screen.getFocused() == null` was added here in 9a5ccb54e51dd19e6c87df79e0794ea4dd46c424 to prevent the captchalogue key from being unintentionally triggered in screens with a text field, such as anvil screens. This need to be adressed beforeyou can remove that check.